### ISSUE TITLE
[BUG] Open Participants list produces crash

### ISSUE THREAD

**Issue by DavidsBees** [cobudget/cobudget#983]
**Describe the bug**
"Unexpected Error" message when trying to access participants page of a round

**To Reproduce**

1. Go to this round: https://cobudget.com/greaterthan/hike-3
2. Click on "Participants"
3. Error

**Expected behavior**
Access participants list

**Browser and device (please complete the following information):**

- Browser: Brave
- Device and/or OS: Windows

**Comment by aerugo** [comment]
I can't reproduce this @DavidsBees. Some more information:
* Were you logged in?
* Are you admin of the round?

**Comment by DavidsBees** [comment]
Yes @aerugo : I was logged in
I am admin of the group but not of the round

**Comment by alinauroz** [comment]
Steps to reproduce the bug

1. Admin of a group create a round
2. Admin of a group (also the admin of the round), make a new admin
3. Any admin remove the group admin as round admin

After step 3, when group admin open the participants page of the round, the page crashes

**Comment by alinauroz** [comment]
Duplicate of https://github.com/cobudget/cobudget/issues/977
